src/screens/GroupSetting/hooks/useGroupSetting.js

```diff
@@ -41,6 +35,7 @@ const useGroupSetting = ({route}) => {
   const [urlImage, setUrlImage] = React.useState(channel?.data?.image);
   const [isLoading, setIsLoading] = React.useState(false);
   const [chat, setChat] = useRecoilState(chatAtom);
+  const {uploadPhotoImage} = useUploadImage();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `uploadPhotoImage` function from the `useUploadImage` hook is now being used. This change seems appropriate as it aligns with the PR title indicating that a method for uploading compressed images has been added.
```diff
+ const {uploadPhotoImage} = useUploadImage();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->